windows/src/buildtools/devtools/devtools.dproj

```diff
@@ -78,7 +78,7 @@
         <DCC_RemoteDebug>true</DCC_RemoteDebug>
     </PropertyGroup>
     <PropertyGroup Condition="'$(Cfg_1_Win32)'!=''">
-        <Debugger_CWD>C:\Projects\keyman\apps\windows\src\ext\dcpcrypt</Debugger_CWD>
+        <Debugger_CWD></Debugger_CWD>
```

Comment:
Oh good, eliminates an absolute path.

Reply:
Mind you, it's only a Debugger setting so it's not important (this should be in a .user file, Delphi!)